playwright/tests/establishment/establishmentWorkflow.spec.ts

```diff
@@ -1,51 +1,72 @@
 import { faker } from "@faker-js/faker/locale/fr";
-import { Page, expect, test } from "@playwright/test";
+import { expect, test } from "@playwright/test";
 import { addMonths } from "date-fns";
-import {
-  FormEstablishmentDto,
-  domElementIds,
-  frontRoutes,
-  toDisplayedDate,
-} from "shared";
+import { FormEstablishmentDto, domElementIds } from "shared";
 import { testConfig } from "../../custom.config";
-import { goToAdminTab } from "../../utils/admin";
+import { phoneRegexp } from "../../utils/utils";
+import { createNewEstablishment } from "./createNewEstablishment";
+import {
+  EstablishmentsRetries,
+  fillEstablishmentFormFirstStep,
+} from "./establishmentForm.utils";
+import { goToManageEtablishmentBySiretInAdmin as goToManageEtablishmentInBackOfficeAdmin } from "./establishmentNavigation.utils";
 import {
-  expectLocatorToBeVisibleAndEnabled,
-  fillAutocomplete,
-  phoneRegexp,
-} from "../../utils/utils";
+  checkAvailabilityThoughBackOfficeAdmin,
+  checkAvailabilityThoughEstablishmentDashboard,
+  checkEstablishmentInAdmin,
+  deleteEstablishmentInBackOfficeAdmin,
+} from "./manageEstablishment";
+import {
+  modifyEstablishmentMagicLink,
+  updateEstablishmentBackOfficeAdmin,
+} from "./modifyEstablishment";
+import { searchEstablishment } from "./searchEstablishment";
 
 test.describe.configure({ mode: "serial" });
-const establishments = [
-  {
-    siret: "41433740200039",
-    expectedAddress: "Avenue des Grands Crus 26600 Tain-l'Hermitage",
-  },
-  {
-    siret: "21590350100017",
-    expectedAddress: "Place Augustin Laurent 59000 Lille",
-  },
-  {
-    siret: "21310555400017",
-    expectedAddress: "1 Place du Capitole 31000 Toulouse",
-  },
-];
 
 test.describe("Establishment creation and modification workflow", () => {
-  const updatedInformations = {
+  const establishmentRetries: EstablishmentsRetries = [
```

Comment:
```suggestion
  const testEstablishments: TestEstablishments = [
```
Je comprenais pas le Retries (je m'attendais à un nombre)